### PR TITLE
fix:ui spacing and dark mode badge design

### PR DESCRIPTION
**Pull request by aayush105**
### Summary
This PR refines the UI by adjusting spacing and enhancing the appearance of badges for dark mode. It addresses visual consistency by minimizing unnecessary gaps, aligning sections, and improving badge readability in dark mode.

### Changes Made
1. **Removed Unnecessary Spacing**: Reduced excess spacing between sections for a cleaner, more compact layout.
2. **Dark Mode Badge Updates**:
   - Modified badge text color for better contrast in dark mode.
   - Adjusted badge background color to ensure readability in both light and dark modes.
3. **Fixed Section Gaps**: Updated gapping between sections to improve consistency.

#### Before:
Screenshot 2024-10-24 161511

Screenshot 2024-10-25 125652

Screenshot 2024-10-25 125659

#### After:
Screenshot 2024-10-24 161531

Screenshot 2024-10-25 125447

Screenshot 2024-10-25 125453

### CODE REVIEW

**Comment by utsavdotdev** [comment]
Hi @aayush105, thanks for raising the PR! It would be best if you first created an issue before raising the PR, as this is the standard process for contributions. No worries! Could you also resolve the alignment issue on larger screens, as shown in the image below?

Screenshot 2024-10-25 at 4 52 13 PM

**Review by utsavdotdev — CHANGES_REQUESTED** [review]
In the event section,  the RSVP should not be in outline

File: src/sections/Events.jsx [hunk 1]
```diff
@@ -221,7 +305,7 @@ const Events = () => {
                     </TabsContent>
                   </Tabs>
                   <div className="mt-4 flex gap-2">
-                    <Button>
+                    <Button variant="outline">
```
Comment (utsavdotdev) [inline src/sections/Events.jsx:308]: This button should be as default not outline

**Comment by aayush105** [comment]
> Hi @aayush105, thanks for raising the PR! It would be best if you first created an issue before raising the PR, as this is the standard process for contributions. No worries! Could you also resolve the alignment issue on larger screens, as shown in the image below?
> 
> Screenshot 2024-10-25 at 4 52 13 PM

Sure, I will fix all the issue and let you know.  @utsavdotdev

**Comment by utsavdotdev** [comment]
Ok

**Comment by aayush105** [comment]
> Ok

Can you review it? I have made the changes @utsavdotdev

**Comment by utsavdotdev** [comment]
Great work @aayush105 , and Congratulation on your first contribution in GNOME Nepal

**Comment by aayush105** [comment]
> Great work @aayush105 , and Congratulation on your first contribution in GNOME Nepal

Thank you broda @utsavdotdev

**Pull request merged**